### DISCUSSION TITLE
Test Error with make_kreport.py

### DISCUSSION THREAD

**Discussion by ryanwelch009** [CDCgov/phoenix#171] [Q&A]
**I am receiving the following error. I am using the following command:**

**nextflow run /home/ryan.welch/phoenix/main.nf -profile test,singularity -entry PHOENIX --kraken2db /data/BI_data/arln/database --singularity_pull_docker_container**

**I am receiving the following error:**
Error executing process > 'PHOENIX:PHOENIX_EXTERNAL:KRAKEN2_WTASMBLD:KRAKENTOOLS_MAKEKREPORT (Test_Sample)'

Caused by:
  Process `PHOENIX:PHOENIX_EXTERNAL:KRAKEN2_WTASMBLD:KRAKENTOOLS_MAKEKREPORT (Test_Sample)` terminated with an error exit status (1)

Command executed:

  make_kreport.py \
      --input Test_Sample.kraken2_wtasmbld.classifiedreads.txt \
      --output Test_Sample.kraken2_wtasmbld.summary.txt \
      --taxonomy database_folder/ktaxonomy.tsv \
      --use-read-len

  cat <<-END_VERSIONS > versions.yml
  "PHOENIX:PHOENIX_EXTERNAL:KRAKEN2_WTASMBLD:KRAKENTOOLS_MAKEKREPORT":
      python: $(python --version | sed 's/Python //g')
      krakentools: 1.2
      phoenix_base_container_tag: base_v2.1.0
      phoenix_base_container: sha256:f0304fe170ee359efd2073dcdb4666dddb96ea0b79441b1d2cb1ddc794de4943
  END_VERSIONS

Command exit status:
  1

Command output:
  PROGRAM START TIME: 07-01-2024 12:56:11
  >> STEP 1/4: Reading taxonomy database_folder/ktaxonomy.tsv...
        0 nodes saved

Command error:
  INFO:    Converting SIF file to temporary sandbox...
  Traceback (most recent call last):
    File "/home/ryan.welch/phoenix/bin/make_kreport.py", line 199, in <module>
      main()
    File "/home/ryan.welch/phoenix/bin/make_kreport.py", line 88, in main
      t_file = open(args.tax_file,'r')
  FileNotFoundError: [Errno 2] No such file or directory: 'database_folder/ktaxonomy.tsv'
  INFO:    Cleaning up image...

Work dir:
  /home/ryan.welch/results/work/b1/50d89b237a225dc9a39dff88de29b0

**Comment by ryanwelch009** [comment]
I could fix this error by setting my --kraken2db $PATH in my home directory. Please disregard. 
RW

**Comment by jvhagey** [comment]
Hi @ryanwelch009, glad you got this fixed next time you have an issue to report, please open an issue here https://github.com/CDCgov/phoenix/issues so it can be triaged.
